core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-store/dot-pages.store.ts

```diff
@@ -570,6 +562,14 @@ export class DotPageStore extends ComponentStore<DotPagesState> {
             });
         }
 
+        // Adding Add To Bundle action
+        if (isEnterprise) {
+            selectItems.push({
+                label: this.dotMessageService.get('contenttypes.content.add_to_bundle'),
+                command: () => this.showAddToBundle(item.identifier)
```

Comment:
test that execute `showAddToBundle ` ?

Reply:
https://github.com/dotCMS/core/blob/913fb818be2e6e6e05a69105b9aafc5fa7a30b94/core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-store/dot-pages.store.spec.ts#L241

already done, this PR only changes the order of item appearence (look and feel), so the functionality and test already exist and no changes were made